src/createTree.js

```diff
@@ -11,23 +11,32 @@ import {asc} from 'num-sort';
  */
 export function createTree(spectrum, options = {}) {
     var X = spectrum[0];
-    const {
+    var {
         minWindow = 0.16,
         threshold = 0.01,
         from = X[0],
         to = X[X.length - 1]
     } = options;
 
-    return mainCreateTree(spectrum[0], spectrum[1], from, to, minWindow, threshold);
+    if (X[0] > X[1]) {
+        if (from < to) [from, to] = [to, from];
+    } else if (from > to) {
```

Comment:
everything can be written in a single condition, was splited with any purpose?

Reply:
I mean with descendant data when the delta of x axis is negative, before, the package return null if the argument from > to or 'deltaX' < 0 and usually the NMR data has 'deltaX' < 0. It change is just for be able to create the tree without reverse the data. Now, is it ok for you? or do you prefer reverse the data.

Reply:
no, it's ok for me, I just didn't understand the idea :+1: